pyproject.toml

```diff
@@ -41,12 +41,13 @@ tune = "torchtune._cli.tune:main"
 dev = [
     "bitsandbytes>=0.43.0",
     "pre-commit",
-    "pytest",
+    "pytest==7.4.0",
```

Comment:
`from torch.testing._internal.common_utils import run_tests` has a depdency on `pytest==7.4.0` and `expecttest`, borrowed from pytorch repo

Reply:
Is `run_tests` strictly required for the usage of `FSDPTest`, or is it more used for convenience? (Either way not a huge issue)

Reply:
it's strictly required for the usage of `FSDPTest`